plugin.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "tesseract-ocr",
-    "version": "2.1.1",
+    "version": "2.1.2",
```

Comment:
[Final nitpick] This could be a new minor version, as we are adding a new feature. So go for 2.2.0